src/components/CommentsCard.jsx

```diff
@@ -0,0 +1,19 @@
+import { Link } from 'react-router-dom';
+import { getTimeAgo } from '../util/format-date.js';
+import Votes from './Votes.jsx';
+import '../css/CommentsCard.css';
+
+export default function CommentsCard({ comment_id, author, created_at, body, votes }) {
+    return (
+        <div key={comment_id} className="comments-card">
```

Comment:
Try and avoid using divs where possible, can we think of a more semantic html tag?